sushibar-backend-kafka/src/main/resources/application.yml

```diff
@@ -28,5 +28,9 @@ spring:
 ## application related properties
 application:
   topic:
-    name: orders
-    partitions: 6
+    producer:
```

Comment:
Why 2 names to refer to the same topic?

Reply:
for now we do not have the consumer topic for the main app. Once the cook MS is ready with its producer topic. which in turn will be consumed by main app then can update the configs. Rest of the code will be in place already.